src/FileManager/FileView.vue

```diff
@@ -38,7 +38,8 @@
                     v-if="isImage"
                     :height="72"
                     :src="`${file.preview_path}?height=80`"
-                    class="mt-4"
+                    class="mt-4 clickable"
+                    style="cursor: pointer"
```

Comment:
Why and a clickable class and `cursor: pointer`
I think you should remove the `style` and add a global class for the clickable

Reply:
`clickable` is already a global-classes, but it issn't recognized. The global-classe is already defined in `src/FileManager/ContextMenu.vue`...
I already tried to place the global-classe in `src/FileManager/FielView.vue`, but that didn't work.